handlers/main.yml

```diff
@@ -1,9 +1,20 @@
 ---
 - name: restart nsqd
-  service: name=nsqd state=restarted enabled=yes
+  supervisorctl:
```

Comment:
should duplicate those handlers and mark them for supervisor specific - do not want to break the comptiility with ubuntu